pages/api/studentPortal/web/react/components.ts

```diff
@@ -0,0 +1,97 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { LessonComponentData } from "../../../../../components/studentPortal/lessons/LessonComponent";
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
I would create a folder called reactjs inside the `pages/api/studentPortal/lessons` folder and move this file into there.

https://github.com/skillify-ca/skillify-web/tree/main/pages/api/studentPortal/lessons/basics